backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -1,17 +1,16 @@
-// Copyright 2021 The Kubeflow Authors
+// Copyright 2021-2023 The Kubeflow Authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      https://www.apache.org/licenses/LICENSE-2.0
+//	https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
NIT: revert?

Reply:
Thank you, I will revert this. I'm not sure why these white spaces were replaced though.

Reply:
Looks like it is caused by VSCode's formatting tools:
https://github.com/microsoft/vscode-go/issues/1930